### DISCUSSION TITLE
Mimicking a person sensor inside the Lovelace Dashboard

### DISCUSSION THREAD

**Discussion by mbrennan** [zachowj/hass-node-red#124] [Q&A]
When I add a person to a card in the Lovelace Dashboard, it shows their location as "Home" or "Away," but if you look at the actual state values of a person entity it's "home" or "not_home" or "<location>" if you have another zone setup.

I created a sensor to mimics the state of a house guest.  I set its value to "home" or "away" but when I add this to the dashboard it shows the actual state value.  Is there some way to mimic a person entity by setting the device_class or state_class?   Or perhaps one of the "Attribute Key" fields?

**Comment by zachowj** [comment]
binary sensor with device class set to presence

**Reply by mikeynohacs** [reply]
Beautiful, thanks @zachowj !  That worked perfectly.  For my own edification, is there some place that I could find more info on the person entities to determine that the sensor would need to be a binary sensor with device class set to presence?  I can't find any reference information about device classes or state classes.

**Reply by zachowj** [reply]
https://www.home-assistant.io/integrations/binary_sensor/
https://developers.home-assistant.io/docs/core/entity/sensor/
